_rules/SC2-4-2-page-has-title.md

```diff
@@ -22,19 +22,19 @@ authors:
 
 ### Applicability
 
-The rule applies to any page where the root element is an `html` element, and which is not embedded in another page.
+The rule applies to any page where the [root](https://www.w3.org/TR/dom41/#concept-tree-root) element is an `html` element, and which is not embedded in another page.
 
 **Note**: Pages may be embedded inside other pages through elements such as iframes and object elements.
 
 ### Expectation 1
 
-The page contains at least one `title` element.
+The [root](https://www.w3.org/TR/dom41/#concept-tree-root) element has at least one [descendant](https://www.w3.org/TR/dom41/#concept-tree-descendant) that is an HTML `title` element.
 
-**Note**: The `title` element exists in other namespaces such as SVG. These are not `title` elements for HTML document and should be ignored.
+**Note**: The `title` element exists in other namespaces such as SVG. These are not HTML `title` elements and should be ignored for this rule.
 
 ### Expectation 2
 
-The first `title` element contains [non-empty text](#non-empty).
+The first `title` element contains [letters or numbers](#letters-or-numbers).
```

Comment:
Just to make sure I understand, this means that the following title would fail, because it only uses punctuation symbols, not letters or numbers?
`<title>#$@&%*!</title>`
And should it fail?

Reply:
@carlosapaduarte, yes, with the current definition it would. The old "non-empty" definition was actually also only looking for letters and numbers, it was just much less obvious. 

I have struggled with the same question for headings. If I create a page about smileys, I would say that ":-)" and ":'-(" would be fine as headings. 

What do you think? Another definition that I have used in some places is "contains text nodes that do not only consist of [Unicode separator characters](https://www.unicode.org/versions/Unicode11.0.0/ch04.pdf#G134153)"
So `<title>#$@&%*!</title>` would be a Passed outcome, but `<title> </title>`would be Failed.

Do you think that would be more appropriate?

We do also have another rule that checks if the title is descriptive, so I think I am actually leaning towards this approach...

Reply:
@annethyme, yes I do prefer the "contains text nodes that do not only consist of Unicode separator characters" definition. 

I don't think that both the `#$@&%*!` title and the `:-)` heading are accessibility violations, so they shouldn't be failed by a rule that checks just a syntactic aspect (in this case, if the page has a title or not).

Reply:
Change has been made and test cases updated to reflect this.